### ISSUE TITLE
Tasks: Dependency 'grpc-google-iam-v1==0.12.0' has syntax error

### ISSUE THREAD

**Issue by plowman** [googleapis/google-cloud-python#8635]
TL;DR: google-cloud-tasks has a dependency on grpc-google-iam-v1. The latest version of grpc-google-iam-v1 (0.12.0) has a syntax error which prevents other packages from importing it.

#### Environment details
1. OS type and version
    * Ubuntu 18.04
2. Python version and virtual environment information: `python --version`
    * Python 3.7.3
3. google-cloud-<service> version:  `pip show google-<service>` or `pip freeze`
```
pip show google-cloud-tasks
Name: google-cloud-tasks
Version: 1.1.0
Summary: Cloud Tasks API API client library
Home-page: https://github.com/GoogleCloudPlatform/google-cloud-python
Author: Google LLC
Author-email: googleapis-packages@google.com
License: Apache 2.0
Location: /fake/env/lib/python3.7/site-packages
Requires: google-api-core, grpc-google-iam-v1
Required-by: 
```

#### Steps to reproduce
  1. Install this requirements.txt:
  ```
  google-api-core[grpc]==1.13.0
  google-cloud-tasks==1.1.0
  grpc-google-iam-v1==0.12.0
  ```
  2. Run this in the python CLI:
  ```
  $ python
Python 3.7.3 (default, Apr  3 2019, 19:16:38) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from google.cloud import tasks_v2beta3
  ```

#### Code example

```python
from google.cloud import tasks_v2beta3
```

#### Stack trace
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/fake/env/lib/python3.7/site-packages/google/cloud/tasks_v2beta3/__init__.py", line 19, in <module>
    from google.cloud.tasks_v2beta3 import types
  File "/fake/env/lib/python3.7/site-packages/google/cloud/tasks_v2beta3/types.py", line 22, in <module>
    from google.cloud.tasks_v2beta3.proto import cloudtasks_pb2
  File "/fake/env/lib/python3.7/site-packages/google/cloud/tasks_v2beta3/proto/cloudtasks_pb2.py", line 25, in <module>
    from google.iam.v1 import iam_policy_pb2 as google_dot_iam_dot_v1_dot_iam__policy__pb2
  File "/fake/env/lib/python3.7/site-packages/google/iam/v1/iam_policy_pb2.py", line 177
    __doc__ = """Request message for ``SetIamPolicy`` method.
            ^
SyntaxError: invalid syntax
```

#### Fix
The fix for this is just to use this style in google/iam/v1/iam_policy_pb2.py:
```
SetIamPolicyRequest = _reflection.GeneratedProtocolMessageType('SetIamPolicyRequest', (_message.Message,), dict(
  DESCRIPTOR = _SETIAMPOLICYREQUEST,
  __module__ = 'google.iam.v1.iam_policy_pb2'
  ,
  __doc__ = """Request message for ``SetIamPolicy`` method.
  
  
  Attributes:
      resource:
          REQUIRED: The resource for which the policy is being
          specified. ``resource`` is usually specified as a path. For
          example, a Project resource is specified as
          ``projects/{project}``.
      policy:
          REQUIRED: The complete policy to be applied to the
          ``resource``. The size of the policy is limited to a few 10s
          of KB. An empty policy is a valid policy but certain Cloud
          Platform services (such as Projects) might reject them.
  """,
  # @@protoc_insertion_point(class_scope:google.iam.v1.SetIamPolicyRequest)
  ))
```

Instead of this:
```
SetIamPolicyRequest = _reflection.GeneratedProtocolMessageType('SetIamPolicyRequest', (_message.Message,), {
  'DESCRIPTOR': _SETIAMPOLICYREQUEST,
  '__module__': 'google.iam.v1.iam_policy_pb2'
  ,
  __doc__ = """Request message for ``SetIamPolicy`` method.


  Attributes:
      resource:
          REQUIRED: The resource for which the policy is being
          specified. See the operation documentation for the appropriate
          value for this field.
      policy:
          REQUIRED: The complete policy to be applied to the
          ``resource``. The size of the policy is limited to a few 10s
          of KB. An empty policy is a valid policy but certain Cloud
          Platform services (such as Projects) might reject them.
  """,
# @@protoc_insertion_point(class_scope:google.iam.v1.SetIamPolicyRequest)
})
```

**Comment by tseaver** [comment]
I can reproduce in a clean virtual environment:

```bash
$ python3.7 -m venv /tmp/gcp/8635
$ /tmp/gcp/8635/bin/pip install --upgrade setuptools pip wheel
...
Successfully installed pip-19.1.1 setuptools-41.0.1 wheel-0.33.4
$ /tmp/gcp/8635/bin/pip install grpc-google-iam-v1==0.12.0
...
Successfully installed googleapis-common-protos-1.6.0 grpc-google-iam-v1-0.12.0 grpcio-1.22.0 protobuf-3.8.0 six-1.12.0
$ /tmp/gcp/8635/bin/python -c "from google.iam.v1 import iam_policy_pb2"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/gcp/8635/lib/python3.7/site-packages/google/iam/v1/iam_policy_pb2.py", line 190
    __doc__ = """Request message for ``SetIamPolicy`` method.
            ^
SyntaxError: invalid syntax
```

**Comment by crwilcox** [comment]
There is a bug in the latest `grpc-google-iam-v1 0.12.x` versions. We are releasing a patch to fix the doc string.

This said, the pinned version of this dependency in `google-cloud-tasks` is `<0.12.0dev`.

Is there a reason you are specifying the dependencies of `google-cloud-tasks`? It is likely safer for you to trust the pinning of the package for both `grpc-google-iam-v1` and `google-api-core`

**Comment by crwilcox** [comment]
v0.12.3 no longer has the syntax error. That said, v0.12.3 is not compatible in every way with v0.11.x versions. Please respect the versions in the setup.py to avoid having similar failures.

**Comment by plowman** [comment]
> Is there a reason you are specifying the dependencies of `google-cloud-tasks`?

@crwilcox yeah, it's a fair question. We have CI which auto-updates all of our packages to the latest version and then runs test/lint, which is how this issue was surfaced to us. This is a pretty good way to make sure everything is up to date while also finding breaking changes as they are published rather than waiting until you are forced to upgrade a package.

Obviously, `google-cloud-tasks` is doing the right thing here by not using the latest version of `grpc-google-iam-v1`, but I figured it was still worth reporting the breaking change in the grpc package.

Thanks a bunch to you and @tseaver for jumping on this so quickly! 😻

**Comment by crwilcox** [comment]
@plowman thanks for escalating this. It helped us fix this early before we took a dependency on it.
